ansible_collections/arista/avd/docs/templates/plugin-docs.j2

```diff
@@ -37,7 +37,7 @@ title: {{ collection | default("arista.avd", true) ~ "." ~ module }}
 {%             set _ = restrictions.append("Max value: <code>" ~ spec.max ~ "</code>") %}
 {%         endif %}
 {%         set description = spec.description | map("md_ify") | map("replace", "\.", ".") | map("replace", "\,", ",") | join("<br>") %}
-| {{ "    " * level }}{{ name }} | {{ typ }} | {{ req }} | {{ def_val }} | {{ restrictions | join("<br>") }} | {{ description }} |
+| <samp>{{ " " * 4 * level }}{{ name }}</samp> | {{ typ }} | {{ req }} | {{ def_val }} | {{ restrictions | join("<br>") }} | {{ description }} |
```

Comment:
You have to use `&nbsp;` instead of space. Otherwise it will just render a single space.

Reply:
Yeah .. :) just wanted to tests hehe